app/Models/Account.php

```
@@ -67,4 +72,10 @@ public function scopeWithAccountName($query, $accountName)
     {
         return $query->where('account_name', $accountName)->first()->id;
     }
+
+    public function scopeTaxable($query, bool $state = true)
```

PHPMD (2.15.0)

The method scopeTaxable has a boolean flag argument $state, which is a certain sign of a Single Responsibility Principle violation. (Clean Code Rules)

(BooleanArgumentFlag)